docker-compose.yml

```diff
@@ -1,13 +1,13 @@
-version: '2'
+version: '3'
 
 services:
   web:
-    image: datacite/viringo
+    image: frdr-dfdr/viringo
```

Comment:
Now that I'm finally looking at this, I think this is the one line that would bring us into conflict with upstream and wouldn't be likely to be merged -- can we change this with an ansible variable or something?